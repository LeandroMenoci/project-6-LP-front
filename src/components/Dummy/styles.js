/*eslint-disable */
import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css``}
`;
