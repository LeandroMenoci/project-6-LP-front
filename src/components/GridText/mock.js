export default {
  background: false,
  title: 'My Grid',
  description:
    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ullam fugiat provident, qui sequi est ad, eos',
  grid: [
    {
      title: 'Teste 1',
      description:
        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?',
    },
    {
      title: 'Teste 2',
      description:
        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?',
    },
    {
      title: 'Teste 3',
      description:
        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, nesciunt! Exercitationem provident est voluptas error, debitis mollitia ipsa voluptatibus optio asperiores praesentium fugit consequatur accusantium nisi, non a nihil sunt?',
    },
  ],
};
