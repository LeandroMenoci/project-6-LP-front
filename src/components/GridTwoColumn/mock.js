export default {
  title: 'Grid tow columns',
  text: `Lorem ipsum dolor sit amet consectetur adipisicing elit. A reprehenderit ex consequatur nesciunt, quo alias ratione consectetur dignissimos, enim magni, odio voluptates voluptatum quaerat ipsa adipisci minima laudantium soluta natus!`,
  srcImg: 'assets/images/javascript.svg',
};
