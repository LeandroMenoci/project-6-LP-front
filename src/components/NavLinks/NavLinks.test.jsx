import { NavLinks } from '.';
import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import mock from './mock';

describe('<NavLinks/>', () => {
  it('Should render links', () => {
    renderTheme(<NavLinks links={mock} />);
    expect(screen.getAllByRole('link')).toHaveLength(mock.length);
  });
});
