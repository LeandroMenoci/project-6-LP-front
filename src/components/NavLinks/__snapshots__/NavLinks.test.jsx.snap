// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavLinks /> should match snapshot 1`] = `
Array [
  <a
    class="sc-gsnTZi laNWyi"
    href="#target1"
    target="_self"
  >
    Link 1
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target2"
    target="_self"
  >
    Link 2
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target3"
    target="_self"
  >
    Link 3
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target4"
    target="_self"
  >
    Link 4
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target5"
    target="_self"
  >
    Link 5
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target6"
    target="_self"
  >
    Link 6
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target7"
    target="_self"
  >
    Link 7
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target8"
    target="_self"
  >
    Link 8
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target9"
    target="_self"
  >
    Link 9
  </a>,
  <a
    class="sc-gsnTZi laNWyi"
    href="#target10"
    target="_self"
  >
    Link 10
  </a>,
]
`;
